### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by vidit09** [fanq15/Few-Shot-Object-Detection-Dataset#7]
Hello,
Could you please elaborate on the process of validation? I couldn't get from the paper what exactly is the validation set for hyperparameter tuning?
Thanks

**Comment by vidit09** [comment]
Thank you, Qi. Adding your reply here for the people with a similar question.

Authors Reply:
The reason we do not set a val set is:

1. The purpose of this dataset is to provide numerous categories to learn relations. So it serves as a base dataset for the few-shot learning tasks on other datasets, e.g., ImageNet and MS COCO. You can treat these datasets as the test set after remove the duplicated categories in our train set.

2. It is really hard to split the dataset into three parts: train, val and test. Because the category relation is very complicated, we have to make sure that similar categories are in the same set. So maybe it is not a good idea to randomly select categories in the train set to validate the model.

Our suggestion is that you can treat the test set as the val set for model design and hyperparameter tuning, and test your model on other datasets. It is also exactly what we do in our paper. We use the test set (maybe it is better to call it val set) to design our model and test it on other datasets.
